core/commonMain/src/kotlinx/serialization/SerializationException.kt

```diff
@@ -62,8 +62,11 @@ public open class SerializationException : IllegalArgumentException {
  * Thrown when [KSerializer] did not receive property from [Decoder], and this property was not optional.
  */
 @PublishedApi
-internal class MissingFieldException(fieldName: String) : // TODO: add (message, cause) ctor after 1.4.20 for coroutines stacktrace recovery
-    SerializationException("Field '$fieldName' is required, but it was missing")
+internal class MissingFieldException : SerializationException {
+    // TODO: add (message, cause) ctor after 1.4.20 for coroutines stacktrace recovery
+    constructor(fieldName: String) : super("Field '$fieldName' is required, but it was missing")
```

Comment:
The error in build is caused by the fact that old plugins can't find primary constructor of this class (because now it has two secondary constructors). In 1.4.20 this problem is fixed, so you may try to bump kotlin version to 1.4.20-M2